README.md

```diff
@@ -129,15 +130,49 @@ cargo fmt --all
 cargo clippy --workspace --all-features -- -D warnings
 ```
 
+## Release Process
+
+### Overview
+
+We make releases within the `rust` repository specific to the crate(s) that will be consumed (e.g. agents@1.0.0, configuration@1.0.0, accumulator@1.0.0, etc).
+
+We follow [Semantic Versioning](https://semver.org/), where breaking changes constitute changes that break agent configuration compatibility.
+
+Releases are managed on GitHub [here](https://github.com/nomad-xyz/rust/releases).
+
+### Aggregating Release Notes
+
+- Want to aggregate list of all changes since last release
+- Run `git diff <tag of last release> HEAD -- **CHANGELOG.md`
```

Comment:
I believe there needs to be a slash `**/CHANGELOG.md`

Reply:
we also probably want `--output=<tag of current release>.changelog.patch` and `--no-prefix`

Reply:
Thought so too but not the case. This one skips changes past singly nested case (e.g. skips agents/kathy/CHANGELOG.md) which confuses me since regex should match. Feel free to try command and see

Reply:
with slash gives me expected output

<img width="712" alt="Screen Shot 2022-06-13 at 11 46 47 AM" src="https://user-images.githubusercontent.com/10149425/173423413-2140bec0-d154-42f7-a7c7-3199415a2cc2.png">

without slash gives me no matches found

<img width="484" alt="Screen Shot 2022-06-13 at 11 46 14 AM" src="https://user-images.githubusercontent.com/10149425/173423327-6a299279-454a-49e7-a5e6-ef638330b761.png">

Reply:
also for technical accuracy, `**/` is a glob, not a regex